recipes/cson/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "cson" %}
+{% set version = "0.7" %}
+{% set sha256 = "c923819ea207e10fd7e58de2920fa621af04be67c307fc2efa36402b253e33a0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - speg
+
+test:
+  imports:
+    - cson
+
+about:
+  home: https://github.com/avakar/pycson
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
`LICENSE` file is not in the `cson-0.7.zip` file, so should I remove the file from the recipe directory and the `license_file` entry in the meta.yaml?